3p/twitter.js

```diff
@@ -16,6 +16,7 @@
 
 // TODO(malteubl) Move somewhere else since this is not an ad.
 
+import {dev} from '../src/log';
```

Comment:
I don't think you can import `log` here, because this is running in a 3p window context. I would say just typecast.

Reply:
Done.